### PR TITLE
Extend self update to include box fixes

### PR DESCRIPTION
**Pull request by bravo-kernel**
Background information in https://github.com/alt3/cakebox/issues/35

### CODE REVIEW

**Comment by ceeram** [comment]
phpcs fails

File: src/Shell/AppShell.php [hunk 1]
```diff
@@ -38,6 +45,7 @@ public function initialize()
         $this->_io->setLoggers(false);
         $this->cbi = new CakeboxInfo();
         $this->execute = new CakeboxExecute();
```
Comment (ceeram) [inline src/Shell/AppShell.php]: Why isnt this removed? you now have 2 instances of cakeboxexecute

File: src/Shell/AppShell.php [hunk 2]
```diff
@@ -28,6 +28,13 @@ class AppShell extends Shell
     protected $execute;
```
Comment (ceeram) [inline src/Shell/AppShell.php]: this should also be removed

File: src/Lib/CakeboxExecute.php [hunk 3]
```diff
@@ -212,7 +171,7 @@ public function composerCreateProject($package, $path)
     {
         $this->_flushLogs();
         $command = "composer create-project --prefer-dist --no-interaction -s dev $package $path";
```
Comment (ceeram) [inline src/Lib/CakeboxExecute.php]: while you are at it, can you include this change as well? #86

**Comment by bravo-kernel** [comment]
Lowercase $this->execute will be removed once all shells are updated. The update i can integrate, np.
I assume the refactoring looks good then :smiley:

**Pull request merged**